### PR TITLE
Chloride two body clean

### PR DESCRIPTION
**Pull request by sqamara**
A cleaned commit history of Implementation of two body force with a  ion, image molecules with an ion and their tests.  Here is the previous pull request of which this is based on https://github.com/paesanilab/mbpol_openmm_plugin/pull/73

### CODE REVIEW

**Comment by sqamara** [comment]
@zonca I've cleaned the two body force commits and closed the old and opened a new pull request

File: platforms/reference/src/MBPolReferenceTwoBodyForce.cpp [hunk 1]
```diff
@@ -22,302 +21,502 @@
  * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
+#include <iostream>
```
Comment (zonca) [inline platforms/reference/src/MBPolReferenceTwoBodyForce.cpp]: we can probably remove iostream

File: platforms/reference/src/MBPolReferenceTwoBodyForce.cpp [hunk 2]
```diff
@@ -22,302 +21,502 @@
...
+			const double d = gsw * dOO[i];
+			forces[allParticleIndices[siteI][0]][i] += d * cal2joule * -10.;
+			forces[allParticleIndices[siteJ][0]][i] -= d * cal2joule * -10.;
+		}
+
+		RealOpenMM energy = sw * E_poly * cal2joule;
+		return energy;
+
+	} else { // if a w-Cl interaction
+		// for the following code, an ion is in siteI so if the ion comes in as siteJ switch the sites
```
Comment (zonca) [inline platforms/reference/src/MBPolReferenceTwoBodyForce.cpp]: love the trick of switching the sites!

File: platforms/reference/tests/TestReferenceMBPolTwoBodyForce.cpp [hunk 3]
```diff
@@ -51,213 +51,597 @@
 
 #define ASSERT_EQUAL_VEC_MOD(expected, found, tol,testname) {ASSERT_EQUAL_TOL_MOD((expected)[0], (found)[0], (tol),(testname)); ASSERT_EQUAL_TOL_MOD((expected)[1], (found)[1], (tol),(testname)); ASSERT_EQUAL_TOL_MOD((expected)[2], (found)[2], (tol),(testname));};
 
-using namespace  OpenMM;
+using namespace OpenMM;
 using namespace MBPolPlugin;
 
-const double TOL = 1e-4;
+const double TOL = 1e-4;\
```
Comment (zonca) [inline platforms/reference/tests/TestReferenceMBPolTwoBodyForce.cpp]: backslash?

**Comment by zonca** [comment]
good job @sqamara , just a couple of cosmetic changes, then this is good to merge

**Comment by sqamara** [comment]
@zonca ready to merge

**Comment by zonca** [comment]
good job!

**Pull request merged**